### PR TITLE
OpenSearch is now supported by The OpenSearch Software Foundation

### PR DESCRIPTION
**Pull request by dblock**
### Description

Minimal content updates that state the basic fact of the OpenSearch Software Foundation.
 
### Issues Resolved

- Part of #216.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by peternied — APPROVED** [review]
Thanks for making these changes along with all the grammar correction.

The only feedback I had was captured with issues you already opened, thanks for being on top of those.

- https://github.com/opensearch-project/.github/issues/219
- https://github.com/opensearch-project/.github/issues/217

File: .github/ISSUE_TEMPLATE/PROPOSAL_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 name: 💭 Proposal
-about: Suggest an idea for a specific feature you wish to propose to the community for comment
+about: Suggest an idea for a specific feature you wish to propose to the community for comment.
```
Comment (CEHENKLE) [inline .github/ISSUE_TEMPLATE/PROPOSAL_TEMPLATE.md:3]: I swear there were periods in here and we went through and took them all out.

File: ONBOARDING.md [hunk 2]
```diff
@@ -1,91 +1,90 @@
...
 
-Any developer who's interested in contributing code (features and fixes) to one of the OpenSearch open source project repositories for the first time.
+Anyone who's interested in contributing code (features, fixes, documentation) to one of the OpenSearch Project open-source repositories for the first time.
 
-### What this guide will help you do
+### What will this guide help you do?
 
-1. Understand the key concepts of the OpenSearch project and how to find additional resources
-2. Set up the accounts you'll need to contribute
+1. Understanding the key concepts of the OpenSearch Project and how to find additional resources.
```
Comment (CEHENKLE) [inline ONBOARDING.md:24]: I think if we're answering the question "what will this guide help you do", then "understanding" is better.  But if you want  change it to understanding, then you should change "set up" to "setting up"?

**Review by CEHENKLE — DISMISSED** [review]

File: .github/workflows/bake_time.yml [hunk 3]
```diff
@@ -11,10 +11,10 @@ on:
 
 jobs:
   baking_pull_request:
-    name: "Baking pull request..."
+    name: "Baking pull request ..."
```
Comment (peternied) [inline .github/workflows/bake_time.yml:14]: Bake time isn't going to count down with this change.  The cron version of this job is running expecting the name to match.

```
No check named 'Baking pull request...' for PR #222. 
```
From GitHub Action run 12771's [logs](https://github.com/opensearch-project/.github/actions/runs/10945551995/job/30390034013)

File: .github/ISSUE_TEMPLATE/PROPOSAL_TEMPLATE.md (hunk 1 above)
Comment (dblock) [reply .github/ISSUE_TEMPLATE/PROPOSAL_TEMPLATE.md:3]: Ok, I'll remove them again.

File: ONBOARDING.md (hunk 2 above)
Comment (dblock) [reply ONBOARDING.md:24]: Put "Understand" back.

File: .github/workflows/bake_time.yml (hunk 3 above)
Comment (dblock) [reply .github/workflows/bake_time.yml:14]: I'll just revert it.

**Review by peternied — APPROVED** [review]

**Comment by dblock** [comment]
@CEHENKLE approve/merge? bake time is done

**Review by CEHENKLE — APPROVED** [review]

**Pull request merged**